src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -386,6 +386,10 @@
                 <source>This value is not a valid International Securities Identification Number (ISIN).</source>
                 <target>Dieser Wert ist keine gültige Internationale Wertpapierkennnummer (ISIN).</target>
             </trans-unit>
+            <trans-unit id="100">
+                <source>This value should be a valid expression.</source>
+                <target>Dieser Wert sollte eine gültige Expression sein.</target>
```

Comment:
We don't translate "expression" because we use it as a technical term here? "Ausdruck" would be a possible alternative.

Reply:
I was considering that as well. Then I decided to treat it as a technical term as it seems it was done the same way die the French translation.

Reply:
FTR: #41326